chainstate/constraints-value-accumulator/src/constraints_accumulator.rs

```diff
@@ -99,39 +96,38 @@ impl ConstrainedValueAccumulator {
                     )?;
                 }
                 TxInput::AccountCommand(_, command) => {
-                    let fee_to_deduct = accumulator.process_input_account_command(
+                    let (id, to_deduct) = accumulator.process_input_account_command(
                         chain_config,
                         block_height,
                         command,
+                        orders_accounting_view,
```

Comment:
Why don't we use the tracker here? I'm not very comfortable with this half and half approach

Reply:
I believe tracker's purpose is to track overspends from accounts across multiple inputs. Here I need orders_view to calculate actual amounts being spend. Am I missing something?

Reply:
the rule that there can only be single Command per tx

Reply:
will be reworked in a separate pr